Makefile

```diff
@@ -56,29 +66,55 @@ docker-from-scratch:
 docker-compile-env:
 	docker build -f build/Dockerfile.interactive-compile-environment -t authservice-build-env:$(USER) .
 
-bazel-bin/src/main/auth_server:
-    # Note: add --compilation_mode=dbg to the end of the next line to build a debug executable with `make docker-from-scratch`
+$(current_binary):
```

Comment:
shall the line below be replaced with `(call bazel-build)`?

Reply:
Updated.